backends-velox/src/main/scala/org/apache/spark/sql/execution/unsafe/UnsafeBytesBufferArray.scala

```diff
@@ -79,7 +75,17 @@ case class UnsafeBytesBufferArray(
     assert(bytesBuffer.length == bytesBufferLengths(index))
     // first to allocate underlying long array
     if (null == longArray && index == 0) {
-      longArray = allocateArray((totalBytes + 7) / 8)
+      val numBytes = totalBytes + 7
```

Comment:
Why the numBytes is not (totalBytes + 7) / 8

Reply:
`allocateArray` accepts number of long elements so number of bytes will be multiplied by 8